#################### ROUND 2: jwang175 (W) vs samana (B): game 1 ####################
##### game start! #####
INPUT for samana:
GAME
BLACK
300
.b.b.b.b
b.b.b.b.
.b.b.b.b
........
........
w.w.w.w.
.w.w.w.w
w.w.w.w.
##### Running: /usr/bin/time -o time_2_jwang175_samana.txt -f "%U" python3 homework.py > stulog.txt 2>&1 on samana #####
child exited with value 1
##### ERROR: /usr/bin/time -o time_2_jwang175_samana.txt -f "%U" python3 homework.py > stulog.txt 2>&1 on samana #####
##### Output of failed program (stdout + stderr): #####
Traceback (most recent call last):
  File "homework.py", line 557, in <module>
    main_start_time =time.clock() #process_time()
AttributeError: module 'time' has no attribute 'clock'
##### End of output of failed program #####
##### samana plays [No Move] in 0.740664958953857s #####
##### No output.txt found. Making a bogus one... #####
##### Running: /usr/lib/jvm/jdk-14.0.1/bin/java -jar /lab/itti/projects/checkers-competition/server/referee.jar on run_2_jwang175_samana #####
Step in a move needs to have 3 sections, found 1 in 'illegal'.
child exited with value 255
##### ERROR: /usr/lib/jvm/jdk-14.0.1/bin/java -jar /lab/itti/projects/checkers-competition/server/referee.jar on run_2_jwang175_samana #####
Game server reported error. Assuming bogus move.
##### GAME OVER: samana loses because of illegal move [No Move] #####
Server said: FAIL
########## ROUND 2 GAMESCORE AFTER GAME 1: jwang175=1 and samana=0 ##########
#################### ROUND 2: jwang175 (B) vs samana (W): game 2 ####################
##### game start! #####
INPUT for jwang175:
GAME
BLACK
300
.b.b.b.b
b.b.b.b.
.b.b.b.b
........
........
w.w.w.w.
.w.w.w.w
w.w.w.w.
##### Running: /usr/bin/time -o time_2_jwang175_samana.txt -f "%U" python3 homework.py > stulog.txt 2>&1 on jwang175 #####
##### SUCCESS: /usr/bin/time -o time_2_jwang175_samana.txt -f "%U" python3 homework.py > stulog.txt 2>&1 on jwang175 #####
##### jwang175 plays [E b6 c5 ] in 2.47s #####
##### Running: /usr/lib/jvm/jdk-14.0.1/bin/java -jar /lab/itti/projects/checkers-competition/server/referee.jar on run_2_jwang175_samana #####
Move is valid
Couldn't read file 'current_ply.txt'
Couldn't read content from file 'current_ply.txt'
Couldn't read file 'lastmatchange_ply.txt'
Couldn't read content from file 'lastmatchange_ply.txt'
##### SUCCESS: /usr/lib/jvm/jdk-14.0.1/bin/java -jar /lab/itti/projects/checkers-competition/server/referee.jar on run_2_jwang175_samana #####
INPUT for samana:
GAME
WHITE
300
.b.b.b.b
b.b.b.b.
...b.b.b
..b.....
........
w.w.w.w.
.w.w.w.w
w.w.w.w.
##### Running: /usr/bin/time -o time_2_jwang175_samana.txt -f "%U" python3 homework.py > stulog.txt 2>&1 on samana #####
child exited with value 1
##### ERROR: /usr/bin/time -o time_2_jwang175_samana.txt -f "%U" python3 homework.py > stulog.txt 2>&1 on samana #####
##### Output of failed program (stdout + stderr): #####
Traceback (most recent call last):
  File "homework.py", line 557, in <module>
    main_start_time =time.clock() #process_time()
AttributeError: module 'time' has no attribute 'clock'
##### End of output of failed program #####
##### samana plays [No Move] in 0.953695774078369s #####
##### No output.txt found. Making a bogus one... #####
##### Running: /usr/lib/jvm/jdk-14.0.1/bin/java -jar /lab/itti/projects/checkers-competition/server/referee.jar on run_2_jwang175_samana #####
Step in a move needs to have 3 sections, found 1 in 'illegal'.
child exited with value 255
##### ERROR: /usr/lib/jvm/jdk-14.0.1/bin/java -jar /lab/itti/projects/checkers-competition/server/referee.jar on run_2_jwang175_samana #####
Game server reported error. Assuming bogus move.
##### GAME OVER: samana loses because of illegal move [No Move] #####
Server said: FAIL
########## ROUND 2 GAMESCORE AFTER GAME 2: jwang175=2 and samana=0 ##########
#################### ROUND 2: jwang175 (W) vs samana (B): game 3 ####################
##### game start! #####
INPUT for samana:
GAME
BLACK
300
.b.b.b.b
b.b.b.b.
.b.b.b.b
........
........
w.w.w.w.
.w.w.w.w
w.w.w.w.
##### Running: /usr/bin/time -o time_2_jwang175_samana.txt -f "%U" python3 homework.py > stulog.txt 2>&1 on samana #####
child exited with value 1
##### ERROR: /usr/bin/time -o time_2_jwang175_samana.txt -f "%U" python3 homework.py > stulog.txt 2>&1 on samana #####
##### Output of failed program (stdout + stderr): #####
Traceback (most recent call last):
  File "homework.py", line 557, in <module>
    main_start_time =time.clock() #process_time()
AttributeError: module 'time' has no attribute 'clock'
##### End of output of failed program #####
##### samana plays [No Move] in 0.715890169143677s #####
##### No output.txt found. Making a bogus one... #####
##### Running: /usr/lib/jvm/jdk-14.0.1/bin/java -jar /lab/itti/projects/checkers-competition/server/referee.jar on run_2_jwang175_samana #####
Step in a move needs to have 3 sections, found 1 in 'illegal'.
child exited with value 255
##### ERROR: /usr/lib/jvm/jdk-14.0.1/bin/java -jar /lab/itti/projects/checkers-competition/server/referee.jar on run_2_jwang175_samana #####
Game server reported error. Assuming bogus move.
##### GAME OVER: samana loses because of illegal move [No Move] #####
Server said: FAIL
########## ROUND 2 GAMESCORE AFTER GAME 3: jwang175=3 and samana=0 ##########
#################### ROUND 2: jwang175 (B) vs samana (W): game 4 ####################
##### game start! #####
INPUT for jwang175:
GAME
BLACK
300
.b.b.b.b
b.b.b.b.
.b.b.b.b
........
........
w.w.w.w.
.w.w.w.w
w.w.w.w.
##### Running: /usr/bin/time -o time_2_jwang175_samana.txt -f "%U" python3 homework.py > stulog.txt 2>&1 on jwang175 #####
##### SUCCESS: /usr/bin/time -o time_2_jwang175_samana.txt -f "%U" python3 homework.py > stulog.txt 2>&1 on jwang175 #####
##### jwang175 plays [E b6 c5 ] in 2.47s #####
##### Running: /usr/lib/jvm/jdk-14.0.1/bin/java -jar /lab/itti/projects/checkers-competition/server/referee.jar on run_2_jwang175_samana #####
Move is valid
Couldn't read file 'current_ply.txt'
Couldn't read content from file 'current_ply.txt'
Couldn't read file 'lastmatchange_ply.txt'
Couldn't read content from file 'lastmatchange_ply.txt'
##### SUCCESS: /usr/lib/jvm/jdk-14.0.1/bin/java -jar /lab/itti/projects/checkers-competition/server/referee.jar on run_2_jwang175_samana #####
INPUT for samana:
GAME
WHITE
300
.b.b.b.b
b.b.b.b.
...b.b.b
..b.....
........
w.w.w.w.
.w.w.w.w
w.w.w.w.
##### Running: /usr/bin/time -o time_2_jwang175_samana.txt -f "%U" python3 homework.py > stulog.txt 2>&1 on samana #####
child exited with value 1
##### ERROR: /usr/bin/time -o time_2_jwang175_samana.txt -f "%U" python3 homework.py > stulog.txt 2>&1 on samana #####
##### Output of failed program (stdout + stderr): #####
Traceback (most recent call last):
  File "homework.py", line 557, in <module>
    main_start_time =time.clock() #process_time()
AttributeError: module 'time' has no attribute 'clock'
##### End of output of failed program #####
##### samana plays [No Move] in 0.664574146270752s #####
##### No output.txt found. Making a bogus one... #####
##### Running: /usr/lib/jvm/jdk-14.0.1/bin/java -jar /lab/itti/projects/checkers-competition/server/referee.jar on run_2_jwang175_samana #####
Step in a move needs to have 3 sections, found 1 in 'illegal'.
child exited with value 255
##### ERROR: /usr/lib/jvm/jdk-14.0.1/bin/java -jar /lab/itti/projects/checkers-competition/server/referee.jar on run_2_jwang175_samana #####
Game server reported error. Assuming bogus move.
##### GAME OVER: samana loses because of illegal move [No Move] #####
Server said: FAIL
########## ROUND 2 GAMESCORE AFTER GAME 4: jwang175=4 and samana=0 ##########


########## GRAND WINNER ROUND 2 IS jwang175 BY 4 vs 0 GAMESCORE ##########
